back/src/test/java/com/cocktailpick/back/user/controller/UserControllerTest.java

```diff
@@ -96,4 +99,21 @@ void deleteFavorite() throws Exception {
 			.andExpect(status().isNoContent())
 			.andDo(print());
 	}
+
+	@WithMockCustomUser
+	@Test
+	void updateCurrentUser() throws Exception {
+		doNothing().when(userService).updateUser(any(), any());
+
+		UserUpdateRequest userUpdateRequest = new UserUpdateRequest("작은곰");
+
+		mockMvc.perform(RestDocumentationRequestBuilders.put("/api/user/me")
```

Comment:
```suggestion
		mockMvc.perform(put("/api/user/me")
```
UserControllerTest 안에서는 이미 `RestDocumentationRequestBuilders`만을 import해서 사용하고 있어서 굳이 명시하지 않아도 될 것 같아요! :)

Reply:
그렇네요! 제거하겠습니다